src/sysupdate/sysupdate.c

```diff
@@ -670,6 +700,7 @@ static int context_show_version(Context *c, const char *version) {
                                           SD_JSON_BUILD_PAIR_BOOLEAN("installed", FLAGS_SET(us->flags, UPDATE_INSTALLED)),
                                           SD_JSON_BUILD_PAIR_BOOLEAN("obsolete", FLAGS_SET(us->flags, UPDATE_OBSOLETE)),
                                           SD_JSON_BUILD_PAIR_BOOLEAN("protected", FLAGS_SET(us->flags, UPDATE_PROTECTED)),
+                                          SD_JSON_BUILD_PAIR_BOOLEAN("incomplete", FLAGS_SET(us->flags, UPDATE_INCOMPLETE)),
```

Comment:
updatectl needs to be updated to handle this. Something like

```diff
diff --git a/src/sysupdate/updatectl.c b/src/sysupdate/updatectl.c
index 0ef61bf895..49cb375720 100644
--- a/src/sysupdate/updatectl.c
+++ b/src/sysupdate/updatectl.c
@@ -360,6 +360,10 @@ static int parse_describe(sd_bus_message *reply, Version *ret) {
                         assert(sd_json_variant_is_boolean(value));
                         if (sd_json_variant_boolean(value))
                                 v.flags |= UPDATE_PROTECTED;
+                } else if (streq(key, "incomplete")) {
+                        assert(sd_json_variant_is_boolean(value));
+                        if (sd_json_variant_boolean(value))
+                                v.flags |= UPDATE_INCOMPLETE;
                 } else if (streq(key, "changelog-url")) {
                         sd_json_variant *entry;
                         assert(sd_json_variant_is_array(value));
```

Reply:
Good point. I'll address this & rebase once #32363 lands

Reply:
Need to make sure it's documented (and supported?) in sysupdated too.